lib/optimization/table_data.go

```diff
@@ -180,44 +176,8 @@ func (t *TableData) Rows() []map[string]any {
 	return rows
 }
 
-type FqNameOpts struct {
-	BigQueryProjectID   string
-	MsSQLSchemaOverride string
-}
-
-func (t *TableData) ToFqName(kind constants.DestinationKind, escape bool, uppercaseEscNames bool, opts FqNameOpts) string {
```

Comment:
Moved to https://github.com/artie-labs/transfer/pull/443/files#diff-25a5472065a253195427b0d5cca5e99df35797c1291b52cd6687d670a585b500R30